docs/book/driver-deployment/installation.md

```diff
@@ -0,0 +1,226 @@
+# vSphere CSI Driver - Installation
+
+The following steps need to be performed on the k8s node where the vSphere CSI drivers controller will be deployed. We recommend installing vSphere CSI driver on your k8s master node.
+
+## Taint Master Node
+
+Make sure that the node is tainted with `node-role.kubernetes.io/master=:NoSchedule`
```

Comment:
`Make sure that the node is tainted with` -> Can we clarify master node?

Reply:
done